packages/node/test/integration/services/firebase-service.ts

```diff
@@ -0,0 +1,91 @@
+import { Address } from "@counterfactual/common-types";
+import firebase from "firebase";
+import FirebaseServer from "firebase-server";
+
+import {
+  IMessagingService,
+  IStoreService
+} from "../../../src/service-interfaces";
+
+const MESSAGING_SERVER_KEY = "messages";
+const STORE_SERVER_KEY = "store";
```

Comment:
Should pull these values from the environment and be different per deployment

Reply:
This is specifically under the test scope though, and so it'll be the same across deployments.

For services that are actually injected into the Node, those should probably pull this stuff from the environment.